fluent-plugin-irc.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_runtime_dependency "fluentd"
+  s.add_runtime_dependency "fluentd", [">= 0.14.0", "< 2"]
```

Comment:
For simplicity of appearance?
I don't want to omit square brackets.
This is more easy-to-understand notation for me.